tests/test_fft.py

```diff
@@ -5,82 +5,87 @@
 rank = comm.Get_rank()
 size = comm.Get_size()
 # Manually set the GPU to use
-os.environ["CUDA_VISIBLE_DEVICES"] = "%d" % (rank + 1)
+# os.environ["CUDA_VISIBLE_DEVICES"] = "%d" % (rank + 1)
+# No longer needed because JAX distritubed will take care of it
```

Comment:
I used this option to run test with a simple mpirun on my local machine that has several GPUs, agreed that on jean zay not necessary